### PR TITLE
Add feature2

### PR DESCRIPTION
**Pull request by jtconnor**
- feature2 is a simple text file

Tested:
- n/a

### CODE REVIEW

File: feature2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (james-corbalt) [inline feature2.txt]: say hello to something more specific

Comment (james-corbalt) [reply feature2.txt]: perhaps a person or animal?

**Review by james-corbalt — CHANGES_REQUESTED** [review]
nice start, needs more work

File: feature2.txt (hunk 1 above)
Comment (jtconnor) [reply feature2.txt]: updated to an animal

**Comment by jtconnor** [comment]
@james-corbalt take another look?

**Review by james-corbalt — APPROVED** [review]
LGTM

**Pull request merged**